### PR TITLE
feat(config): updated params for LZW31-SN and LZW31

### PR DESCRIPTION
**Pull request by Hilo4321**
correcting requested changes from #1700

### CODE REVIEW

File: packages/config/config/devices/0x031e/lzw31-bsd_0.0-1.45.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
-// Inovelli LZW31-SN
-// Red Series Dimmer
+// Inovelli LZW31
+// Black Series Dimmer
```
Comment (Petro31) [inline packages/config/config/devices/0x031e/lzw31-bsd_0.0-1.45.json:2]: Remove comments to pass lint

File: packages/config/config/devices/0x031e/lzw31-bsd_1.47-1.47.json [hunk 2]
```diff
@@ -0,0 +1,366 @@
+// Inovelli LZW31
+// Black Series Dimmer
```
Comment (Petro31) [inline packages/config/config/devices/0x031e/lzw31-bsd_1.47-1.47.json:2]: Same

File: packages/config/config/devices/0x031e/lzw31-bsd_1.49.json [hunk 3]
```diff
@@ -0,0 +1,377 @@
+// Inovelli LZW31
+// Black Series Dimmer
```
Comment (Petro31) [inline packages/config/config/devices/0x031e/lzw31-bsd_1.49.json:2]: same

File: packages/config/config/devices/0x031e/lzw31-sn_0.0-1.45.json [hunk 4]
```diff
@@ -1,4 +1,4 @@
-// Inovelli LZW31-SN
+// Inovelli LZW31-SN
```
Comment (Petro31) [inline packages/config/config/devices/0x031e/lzw31-sn_0.0-1.45.json:1]: same

File: packages/config/config/devices/0x031e/lzw31-sn_1.47-1.47.json [hunk 5]
```diff
@@ -0,0 +1,409 @@
+// Inovelli LZW31-SN
+// Red Series Dimmer
```
Comment (Petro31) [inline packages/config/config/devices/0x031e/lzw31-sn_1.47-1.47.json:2]: same

File: packages/config/config/devices/0x031e/lzw31-sn_1.49.json [hunk 6]
```diff
@@ -0,0 +1,420 @@
+// Inovelli LZW31-SN
+// Red Series Dimmer
```
Comment (Petro31) [inline packages/config/config/devices/0x031e/lzw31-sn_1.49.json:2]: same

**Review by Petro31 — CHANGES_REQUESTED** [review]
Remove the comments an it will pass lint.  They aren't needed anyways

**Comment by AlCalzone** [comment]
The comments shouldn't affect linting.
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
Hmm guess there was a space at the end

**Comment by Petro31** [comment]
> The comments shouldn't affect linting.
> @zwave-js-bot fix lint

Odd, when I was running lint, that was the only thing that was failing.  My bad!

**Comment by AlCalzone** [comment]
I think this was superseded by #1740, right?

**Comment by Hilo4321** [comment]
> I think this was superseded by #1740, right?

I think these are actually the same.

**Pull request closed without merging**